spec/spec.md

```diff
@@ -24,8 +24,7 @@ the standard response format, so that the `data` property holds the payload.
 ## Success
 Schema: [`Response.schema.json`](Response.schema.json)
 
-Example
-
+### Example
```

Comment:
There are some changes here: The new validator insists that the examples are introduced with a header.